### ISSUE TITLE
How are multiple displays handled?

### ISSUE THREAD

**Issue by LoeizD** [wanderingstan/Lifeslice#16]
I have two monitors and only one screen is appearing in lifeslice browser (the main display). Two screenshots are captured on my hard drive. Is there a way to have both shown?

**Comment by wanderingstan** [comment]
You're right -- currently the screenshot file is saved but is not displayed anywhere. :( Probably could add an option on the report page for "Display #2".
